src/Algebra/Solver/Ring/Simple/AlmostCommutativeRing/Instances.agda

```diff
@@ -0,0 +1,48 @@
+{-# OPTIONS --without-K --safe #-}
+
+module Polynomial.Simple.AlmostCommutativeRing.Instances where
```

Comment:
This module's name doesn't match it's location. We also have special modules already existing for these `Data.Nat.Solver` and `Data.Int.Solver`